xgo.go

```diff
@@ -222,6 +223,14 @@ func checkDockerImage(image string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
+	if strings.Contains(image, ":") {
+		// get repository and tag
+		res := strings.SplitN(image, ":", 2)
+		r, t := res[0], res[1]
+		match, _ := regexp.Match(fmt.Sprintf(`%s\s+%s`, r, t), out)
```

Comment:
Are you sure this will match only the xgo image? Since `out` has all docker images on the host.
Say, I have the xgo image with some version, not the one I search here and another one with the tag I'm looking for here. Does your regex only match an image + tag combination in the same line?

Reply:
yes, I will add some test cases